client/roundCtrl.ts

```diff
@@ -1510,3 +1374,68 @@ export default class RoundController {
         }
     }
 }
+
+/**
+ * Custom variant-specific logic to be triggered on move and alter state of board/pocket depending on variant rules.
+ * todo: contains also some ui logic - maybe good to split pure chess rules (which maybe can go to chess.ts?)
+ *       from rendering dialogs and
+ * */

```

Comment:
This thing has so many TODOs
```suggestion
/**
 * Custom variant-specific logic to be triggered on move and alter state of board/pocket depending on variant rules.
 * TODO: contains also some ui logic - maybe good to split pure chess rules (which maybe can go to chess.ts?)
 *       from rendering dialogs and
 * TODO: Unify this with analysisCtrl
 * */
```

Reply:
I have actually unified this with analysisCtrl i think